### DISCUSSION TITLE
How to use local outdoor temperature sensor?

### DISCUSSION THREAD

**Discussion by Sawadee2u** [HASwitchPlate/HASPone#224] [Q&A]
Prior to the updating of the blueprints for the Haspone I was able to select entities for temp sensors.  Now can only use a weather service provided temperature. Is there a way to select local sensor entities?
Thanks

**Comment by chairstacker** [comment]
I just received my HASPone today this was my first try to get something on the display, so I just 'hacked' the blueprint called 'HASPone p[x].b[y] displays the current temperature from a sensor with icon and colors' by making these changes in the yaml:

I changed line 120 to: 
```domain: sensor```
This allows me to pick a temperature sensor instead of a weather provider from the dropdown.

And in lines 275 and 275 I replaced the reference to the attribute with ```states(weather_provider)```

There are for sure more elegant ways to achieve that, but this was my quick and brutal fix for now.

**Comment by Sawadee2u** [comment]
And in lines 275 and 275 I replaced--------   What?  275 and what?

**Reply by chairstacker** [reply]
Seems like there was a significant change between V1.05 that I'm using, and V1.06 which is online here now.
Looking at my V1.05, I think it would have been 275 and 276, no idea where this would be in the current version - might be 223 - 228.

**Reply by Sawadee2u** [reply]
Thanks. I haven't been able to get temperature to show since an update over a year ago on all three HASPones. Have updated to lasted version and removed the blueprint (all of them) and completely reconfigured with the latest on every blueprint.
All works but the temperature with icons. Just a white bar (1-6) location. Have three weather systems configured and reporting in HA and all have "temperature" with current data. Have given up. So was looking to just use my outside temp from my Davis Weather system which is reported by mqtt to my HA system.
Thanks
